<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rank</class>
 <widget class="QMainWindow" name="rank">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>391</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name1"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="time1"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="name2"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="time2"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="name3"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="time3"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="name4"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="time4"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="name5"/>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="time5"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
